login.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓 - 로그인</title>
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+    <link rel="stylesheet" href="signin.css">
+  </head>
+
+    <body>
+      <header>
+        <div class="bigLogo">
+            <a href="/">
+                <img id="panda"
+                 src="images/logopandal2x.png" alt="판다마켓">
+            </a>
+        </div>
+      </header>
+        <form class=" login">
+          <label>이메일</label>
+          <input
+            id="email"
```

Comment:
P2:
label, input을 연결해주면 label 클릭 시 input으로 focus가 이동됩니다.
일반적으로 유저가 기대할 수 있는 동작이니 연결해주시는 것이 좋습니다.

```suggestion
          <label for='email'>이메일</label>
          <input
            id="email"
```